osf/utils/fields.py

```diff
@@ -12,7 +12,7 @@ class LowercaseCharField(models.CharField):
     def get_prep_value(self, value):
         value = super(models.CharField, self).get_prep_value(value)
         if value is not None:
-            value = value.lower()
+            value = str(value).lower()
```

Comment:
Ok. I see now that `super()` is called before this line, so I assume invalid types are handled there?